cloudscape-design/components
# Aws UI 18910: Splitpanel slider made keybaord accessible when position is on side

### Description
Add https://github.com/cloudscape-design/components/pull/215 back and fix: 
Changes to set offset padding on aside since setting it on parent div was causing extra space over the split panel when visual refresh was disabled

### How has this been tested?

- manual testing with visual-refresh disabled
- npm test

[_Check for unexpected visual regressions, see [`CONTRIBUTING.md`](CONTRIBUTING.md#run-visual-regression-tests) for details._]

### Documentation changes

[*Do the changes include any API documentation changes?*]

### Related Links

[*Attach any related links/pull request for this change*]


<details>
   <summary>Review checklist</summary>

_The following items are to be evaluated by the author(s) and the reviewer(s)._

#### Correctness


#### Security

- [ ] _If the code handles URLs: all URLs are validated through [the `checkSafeUrl` function](https://github.com/cloudscape-design/components/blob/main/src/internal/utils/check-safe-url.ts)._

#### Testing

</details>

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
